frontend/scripts/app/app.component.jsx

```diff
@@ -41,6 +45,11 @@ function App() {
         <CookieBanner />
         {feedback && <Feedback />}
       </main>
+
+      <Suspense fallback={null}>
+        <NewsletterForm />
```

Comment:
This is showing on every page. Even explore and tool. I think it's only supposed to go on the landing page. Isn't it?

Reply:
🤦 good catch

Reply:
fixed